### DISCUSSION TITLE
请问下为什么现在官方文档有很多广告？

### DISCUSSION THREAD

**Discussion by yugasun** [vueComponent/ant-design-vue#6701] [Q&A] [answered]
当前 [官方文档](https://antdv.com/components/overview) 引入了很多广告，非常影响体验。如下：
<img width="800" alt="image" src="https://github.com/vueComponent/ant-design-vue/assets/8536173/84167eed-e9a8-4bbf-88bb-5d785cf3ac8a">
一个页面竟然出现3处，特别是下方弹出广告，出现频率特别高。

最不能忍受的是，点击跳转链接还会出现弹层广告，如下：
<img width="800" alt="image" src="https://github.com/vueComponent/ant-design-vue/assets/8536173/3f202c4b-af30-43e0-8b66-c879e6b08f14">

**Comment by tangjinzhou** [comment, marked as answer]
开了 google 自动广告，已关闭
